src/Arcus.WebApi.Correlation/IApplicationBuilderExtensions.cs

```diff
@@ -12,7 +12,7 @@ public static class IApplicationBuilderExtensions
         /// <summary>
         /// Adds operation and transaction correlation to the application by using the <see cref="CorrelationMiddleware"/> in the request pipeline.
         /// </summary>
-        public static IApplicationBuilder UseCorrelation(this IApplicationBuilder applicationBuilder)
+        public static IApplicationBuilder UseHttpCorrelation(this IApplicationBuilder applicationBuilder)
```

Comment:
No need to change this because it's a breaking change and we're building an HTTP Web API

Reply:
But I can see the point of it. If we use this name, we have to provide the old one and deprecate it.

Reply:
Done.